### DISCUSSION TITLE
How do the exclusion settings work?

### DISCUSSION THREAD

**Discussion by 13strannik** [brianpetro/obsidian-smart-connections#107] [General]
I would like to clarify:
- as far as I understand, after the start of adding files, if certain folders are excluded, then the added files remain, they are not deleted?
- how to spell unincluded folders correctly?: "folder1", "folder2" or folder1, folder2
- if you run "Forse Refresh", then all information about additions is deleted and the whole process starts again? Including folders not included in this process?
- the "embeddings" file is twice as large as the entire storage, what is the reason for this?

**Comment by brianpetro** [comment]
> as far as I understand, after the start of adding files, if certain folders are excluded, then the added files remain, they are not deleted?

Correct.

> how to spell unincluded folders correctly?: "folder1", "folder2" or folder1, folder2

folder1, folder2

> if you run "Forse Refresh", then all information about additions is deleted and the whole process starts again? Including folders not included in this process?

It's like starting from the beginning. But your settings will remain. So if you excluded files, they will not be included after the force refresh. 

> the "embeddings" file is twice as large as the entire storage, what is the reason for this?

Each embedding is >15,000 characters. Some notes will have two embeddings, sometimes even more. But yes, the embeddings file gets quite large rather quickly.

**Comment by huachuman** [comment]
I'm having issues with excluded files. I'm getting a lot of notes for which it says "excluded" but I have everything blank in the options. there shouldn't be anything excluded.

**Comment by john-crouch** [comment]
For anyone else that runs into this, I restarted the plugin and that cleared up the issue.
